src/generators/javascript/templates/index.hbs

```diff
@@ -151,7 +151,9 @@ function validateAgainstSchema(
 			allErrors: true,
 			verbose: true
 		})
-		ajv.addMetaSchema(require('ajv/lib/refs/json-schema-draft-06.json'))
+		import('ajv/lib/refs/json-schema-draft-06.json').then((module) => {
```

Comment:
Hi!  As discussed with Siva Shanmuck Vetcha, this dynamic import is asynchronous, so it will behave a bit differently than the `require`, which is synchronous (afaik).  I haven't dug into the code, but I'm guessing you might need to wait for it before calling `ajv.validate(schema, message)` on line 158 or you could end up with a race condition.

Reply:
Hey, thanks for your feedback. We will make a refactor to fix this potential issue